__mocks__/got.js

```diff
@@ -0,0 +1,73 @@
+const people = require('./../assets/people.json');
```

Comment:
By requiring these for the entire module, instead of just as you need them, you're bringing in way more data than you need, which is going to impact memory overhead while running tests.